src/main/scala/tech/cryptonomic/conseil/util/ConfigUtil.scala

```diff
@@ -0,0 +1,61 @@
+package tech.cryptonomic.conseil.util
+
+import com.typesafe.config.ConfigFactory
+import scala.collection.JavaConverters._
+
+import java.util.HashMap
+
+object ConfigUtil {
+
+  private val conf = ConfigFactory.load
+
+  /**
+    * Output Map, keys being blockchain platforms, values being comma and space
+    * separated string of their networks
+    * @return platform, network Map
+    */
+  def parsePlatformsAndNetworks(): Map[String, String] = {
```

Comment:
Will the result change over different calls?

Otherwise it could be possible to make it a [lazy] `val` and avoid recomputing it everytime.

That will also simplify the following method on like `39`
```scala
  def parseNetworksForAGivenPlatform(platform: String): String =
    parsePlatformsAndNetworks(platform)
```